.github/workflows/ci.yml

```
@@ -19,6 +19,27 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check-asset-inventory-supported-version:
+    name: Check Asset Inventory supported version
+    runs-on: ubuntu-22.04
+    outputs:
+      asset_inventory_supported: ${{ steps.check-asset-inventory-supported-version.outputs.asset_inventory_supported }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Check Asset Inventory supported version
+        id: check-asset-inventory-supported-version
+        run: |
+          MIN_VERSION="8.16.0"
+          if [[ "$(echo -e "$MIN_VERSION\n${{ env.ELK_VERSION }}" | sort -V | head -n 1)" == "$MIN_VERSION" ]]; then
+            echo "Stack version meets the requirement: ${{ env.ELK_VERSION }} >= $MIN_VERSION."
+            echo "asset_inventory_supported=true" >> $GITHUB_ENV
+          else
+            echo "Stack version is below the requirement: ${{ env.ELK_VERSION }} < $MIN_VERSION."
+            echo "asset_inventory_supported=false" >> $GITHUB_ENV
+          fi
```

Please use an intermediate environment variable instead of directly using `env.ELK_VERSION`
```suggestion
      - name: Check Asset Inventory supported version
        id: check-asset-inventory-supported-version
        env:
          STACK_VERSION: ${{ env.ELK_VERSION }}
        run: |
          MIN_VERSION="8.16.0"
          if [[ "$(echo -e "$MIN_VERSION\n$STACK_VERSION" | sort -V | head -n 1)" == "$MIN_VERSION" ]]; then
            echo "Stack version meets the requirement: $STACK_VERSION >= $MIN_VERSION."
            echo "asset_inventory_supported=true" >> $GITHUB_ENV
          else
            echo "Stack version is below the requirement: $STACK_VERSION < $MIN_VERSION."
            echo "asset_inventory_supported=false" >> $GITHUB_ENV
          fi
```